go/vt/binlog/slave_connection.go

```diff
@@ -46,13 +48,23 @@ type SlaveConnection struct {
 	wg      sync.WaitGroup
 }
 
+// We use a random slaveid deprecating IDPool which was causing problems with RDS
+// either because it was using ids in the same range that we generate or because
+// the pool reuses ids.
+func getSlaveID() uint32 {
+	fmt.Printf("In getSlaveId\n")
+	max := big.NewInt(math.MaxInt32)
+	id, err := crand.Int(crand.Reader, max)
```

Comment:
We should avoid panics. In this case, it's better to ignore the error. Worst case scenario is that we'll return 0, which will cause the binlog request to fail.